app/templates/suppliers/view_signed_agreement.html

```diff
@@ -4,24 +4,20 @@
   Countersign {{ framework.name }} agreement for {{ supplier_framework.declaration.nameOfOrganisation }} – Digital Marketplace admin
 {% endblock %}
 
-{% block breadcrumb %}
-    {%
-    with items = [
-        {
-            "link": url_for('.index'),
-            "label": "Admin home"
-        },
-        {
-            "link": url_for('.list_agreements', framework_slug=framework.slug),
-            "label": "%s agreements" | format(framework.name)
-        },
-        ]
-    %}
-        {% include "toolkit/breadcrumb.html" %}
-    {% endwith %}
-{% endblock %}
-
 {% block main_content %}
+  {{ govukBreadcrumbs({
+    "items": [
+      {
+        "text": "Admin home",
+        "href": url_for('.index')
+      },
+      {
+        "text": "%s agreements".format(framework.name),
```

Comment:
Ah, this will need to be something like `"{} agreements".format(framework.name)'`. Otherwise we'll see something like this:

![format-string-breadcrumbs](https://user-images.githubusercontent.com/3492540/64848760-dcdc8500-d609-11e9-97f5-eb6604056260.png)

Reply:
Nice spot 🔍

Reply:
I'll fix this now